packages/session-replay-browser/src/config/types.ts

```diff
@@ -41,6 +46,7 @@ export interface SessionReplayLocalConfig extends Config {
   debugMode?: boolean;
   configEndpointUrl?: string;
   shouldInlineStylesheet?: boolean;
+  interactionConfig?: InteractionConfig;
```

Comment:
I forgot which config interface is external, but ideally we don't allow configuration of this through SDK?